Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by 써니쿠키 on 2022/09/22.
+//
+
+import Foundation
+
+enum ExpressionParser {
+    static func parse(from input: String) throws -> Formula {
+        var formula = Formula()
+        let operands = componentsByOperators(from: input)
+        let operators = input.map { $0 }.compactMap { Operator.init(rawValue: $0) }
```

Comment:
compactMap을 사용하신 부분 아주 좋습니다!!

다만,

```suggestion
        let operators = input.compactMap { Operator.init(rawValue: $0) }
```

두 경우의 결과가 다를까요, 같을까요?

Reply:
- 아! 감사합니다! 수정했습니다! [5ac99b0](https://github.com/yagom-academy/ios-calculator-app/pull/337/commits/5ac99b005df6cb93e6d292fdb0fb8871d0f73e9c)